pkg/ttnpb/field_mask_validation.go

```diff
@@ -313,6 +313,8 @@ var nsEndDeviceReadFieldPaths = [...]string{
 	"mac_state.queued_join_accept.request.selected_mac_version",
 	"mac_state.queued_responses",
 	"mac_state.rx_windows_available",
+	"mac_state.recent_downlinks",
```

Comment:
please make sure this is sorted alphabetically

Reply:
Should I sort only `mac_state` lines only, or the whole slice ? I wanted to do this initially but the diff was large so I thought I'm missing something about the sorting.

Reply:
The whole slice is supposed to be sorted and if it's not - we should fix that

Reply:
Note the case-insensitive sorting!

Reply:
In `vim`: `:sort i`